CMakeLists.txt

```diff
@@ -13,14 +14,18 @@ if(WIN32)
 endif()
 
 find_package(PkgConfig REQUIRED)
+find_package(OpenSSL REQUIRED)
```

Comment:
OpenSSL, just for the sake of md5 hash, is redundant. Can we use a lighter implementation or another algorithm like crc32. Are there any analogs in the standard c++ library?